templates/core/stories.html

```diff
@@ -220,7 +220,7 @@ <h2>Ubuntu Core in digital signage</h2>
       <h3 class="p-heading--4">Screenly innovates IoT fleet management for secure digital signage</h3>
       <p>Originally having relied on home-grown infrastructure to deliver over-the-air (OTA) patches to devices in the field, Screenly needed a better way to manage its fast scaling and distributed IoT fleet. After a comprehensive market evaluation, Ubuntu Core stood out as the ideal fit.</p>
       <p>
-        <a href="/engage/secure-digital-signage-screenly">Screenly innovates IoT fleet management for secure digital signage&nbsp;&rsaquo;</a>
+        <a href="/engage/secure-smart-displays-screenly">Screenly innovates IoT fleet management for secure digital signage&nbsp;&rsaquo;</a>
```

Comment:
This page doesn't exist. Revert to `/engage/secure-digital-signage-screenly`?

Reply:
yes, that link is for the engage page of a case study: https://ubuntu.com/engage/secure-digital-signage-screenly
so we shouldn't change that URL